code/modules/customitems/item_defines.dm

```diff
@@ -3350,3 +3350,179 @@ All custom items with worn sprites must follow the contained sprite system: http
 	icon_state = "ulmari_coat"
 	item_state = "ulmari_coat"
 	contained_sprite = TRUE
+
+
+/obj/item/clothing/suit/storage/toggle/labcoat/fluff/mekesatis_labcoat //Biochemist Holocoat - Neith Mekesatis - vrow
+	name = "biochemist holocoat"
+	desc = "An Eridani Corporate Federation holocoat modelled after a standard biochemist labcoat. It is extremely well cared for."
+	icon = 'icons/obj/custom_items/mekesatis_holocoat.dmi'
+	icon_state = "mekesatis_labcoat"
+	item_state = "mekesatis_labcoat"
+	icon_open = "mekesatis_labcoat_open"
+	icon_closed = "mekesatis_labcoat"
+	contained_sprite = TRUE
+	var/changed = FALSE
+	var/changing = FALSE
+
+/obj/item/clothing/suit/storage/toggle/labcoat/fluff/mekesatis_labcoat/examine(mob/user)
+	..()
+	if(in_range(user, src) && !(all_languages[LANGUAGE_TRADEBAND] in user.languages))
+		to_chat(user, SPAN_NOTICE("On the inside of the coat there are various sentences in Tradeband printed in an elegant blue font."))
+		return
+
+	if(in_range(user, src) && (all_languages[LANGUAGE_TRADEBAND] in user.languages))
+		to_chat(user, SPAN_NOTICE("On the inside of the coat, the following words are printed in an elegant blue font:<br>Exclusive Time Limited Holocoat Deal from July 30, 2459. \
+		Now with graced with an animated Eridani Corporate Federation logo. For the Prosperity of all Eridanians - <i>Delta HoloTextiles. Sector Alpha's best \
+		wears.</i><br><small><i><font face='Courier New'>Every cloud has a silver lining, and you should be happy for yours. Congratulations on your \
+		graduation.</font> - <font face='Times New Roman'>Teremun A. M.</font></i></small>"))
+		return
+
+	else
+		to_chat(user, SPAN_NOTICE("There might be something written on the inside of the coat. You have to get closer if you want to read it."))
```

Comment:
It looks like the in_range check should just be the first check done, with an else to lead to the "there might be something written inside" message, and the rest after in another if/else.